lib/membrane_element_ivf/vp9_to_ivf_filter.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Membrane.Element.IVF.VP9 do
+  @moduledoc false
+  use Membrane.Filter
+  use Membrane.Log
+
+  use Ratio
+  alias Membrane.{Buffer, Time, RemoteStream}
+  alias Membrane.Caps.VP9
+
+  def_options width: [spec: [integer], description: "width of frame"],
+              height: [spec: [integer], description: "height of frame"],
+              scale: [spec: [integer], default: 1, description: "scale"],
+              rate: [spec: [integer], default: 1_000_000, description: "rate"]
+
+  def_input_pad :input,
+    caps: {RemoteStream, content_format: VP9, type: :packetized},
+    demand_unit: :buffers
+
+  def_output_pad :output, caps: :any
+
+  defmodule State do
+    @moduledoc false
+    defstruct [:width, :height, :timebase, framecount: 0, header_sent?: false]
```

Comment:
- framecount is unused
- instead of keeping `header_sent?`, you can check `ctx.pads.output.start_of_stream?`

Reply:
This is resolved in #2 because it requires rewriting tests which I've already done in #2. In that PR I've implemented sort of unit tests so I would have to write my own mocked context structure which is normally passed to callback by membrane_core. In the next PR I'm testing serializer in a small testing pipeline so the context is properly passed to callbacks and it can be used in code.

Reply:
Then maybe let's join these PRs together. The whole profit of having separate PRs is to have reviewed, working parts of code once subsequent PRs are merged. It's possible when the PRs are somehow independent of each other, while the tests are very dependent on the code they cover.